src/Umbraco.Web.UI.Client/src/packages/core/property/components/property-layout/property-layout.element.ts

```diff
@@ -90,8 +90,14 @@ export class UmbPropertyLayoutElement extends UmbLitElement {
 
 	#renderDescription() {
 		if (!this.description) return;
-		const ufmValue = { alias: this.alias, label: this.label, description: this.description };
-		return html`<umb-ufm-render id="description" .markdown=${this.description} .value=${ufmValue}></umb-ufm-render>`;
+
+		let description = this.description;
+		if (description.startsWith('#')) {
+			description = this.localize.string(description);
+		}
```

Comment:
@AndyButland We had some back & forth on whether we should do this, as the property descriptions now support Markdown rendering, where the syntax for a markdown heading would start with a `#` (not that we'd encourage headings in property descriptions). Which may lead to confusion as that would be valid markdown syntax.

The UFM render does support localizing string, but they need to be wrapped in curly braces, e.g. `{#documentTypes_TextBlockDescription }`.

The documentation on this is out of date; we have a task to update it: https://github.com/umbraco/UmbracoDocs/issues/6811

Reply:
Ah, OK - so would it be better if I reverted the parts of this that related to property and document type descriptions, and just keep the part around the name of the block when rendered as a node reference?

Reply:
Yes please. 🙏 

The Block label one should be fine, as the markdown renderer is configured for "inline" only, so it wouldn't be capable of rendering headings, etc.

Reply:
OK, I've reverted that part.